pkg/util/pointer/pointer.go

```diff
@@ -64,6 +64,12 @@ func IntToInt32Ptr(i int) *int32 {
 	return &o
 }
 
+// IntToInt64Ptr takes an int and returns a pointer to an int64
+func IntToInt64Ptr(i int) *int64 {
```

Comment:
I doubt that we need this function. Looks like we can use `utilpointer.Int64Ptr()` here because we always specify values explicitly and we can guarantee that these numbers fall within int64 boundaries.

Reply:
Could you try/check this idea and report back whether it works or not?